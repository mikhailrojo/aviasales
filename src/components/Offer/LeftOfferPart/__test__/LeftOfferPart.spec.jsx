import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {LeftOfferPart} from '../index';

describe('Snapshot', () => {
	it('render LeftOfferPart with the offer price, currency and carrier', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<LeftOfferPart
				price={12000}
				carrier="SU"
				currency="RUB"
			/>
		);
		expect(component).toMatchSnapshot();
	});
});