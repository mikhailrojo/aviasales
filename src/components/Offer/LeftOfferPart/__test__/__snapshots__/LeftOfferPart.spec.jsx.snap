// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render LeftOfferPart with the offer price, currency and carrier 1`] = `
<styled.div>
  <styled.div
    url="SU.jpg"
  />
  <styled.button>
    <div>
      Купить
    </div>
    <div>
      за 12000 ₽
    </div>
  </styled.button>
</styled.div>
`;
